virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-hot-plugging-virtual-disks"]
+= Hot-plugging virtual disks
+include::modules/virt-document-attributes.adoc[]
+:context: virt-hot-plugging-virtual-disks
+
+:FeatureName: Hot-plugging virtual disks
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+
+If you _hot-plug_ a virtual disk, you can attach or detach the virtual disk to or from a virtual machine instance while a virtual machine is running.
+
+When you _hot-unplug_ a virtual disk, you can no longer  attach or detach the virtual disk to or from a virtual machine instance while a virtual machine is running.
+
+Only hot-plugged disks can be attached or detached while a virtual machine is running, as opposed to virtual disks that are specified in a virtual machine definition, which cannot be attached or detached before you stop the virtual machine or virtual machine instance.
+
+Only data volumes and persistent volume claims (PVCs) can be hot-plugged and hot-unplugged. You cannot hot-plug or hot-unplug container disks.
+
+Use hot-plugging and hot-unplugging when:
+
+* You want to add or remove virtual disks to balance your storage capacity without stopping your virtual machine or virtual machine instance.
+* You want to proactively add storage to an {product-title} cluster storage class and ensure that the added storage is transparent to a cluster running {VirtProductName} under {product-title}. In this case, you hot-plug a virtual disk that is defined to a {product-title} storage class.
```

Comment:
This example needs a lot more context if you want it to make sense. This is for the case where you are running Open Shift on Open Shift. Which I don't think is currently supported by RH. We are working on it, but it is not quite there yet. So I would remove this particular example.

Reply:
Deleted.